packages/libs/eda/src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -752,7 +753,8 @@ function ScatterplotViz(props: VisualizationProps<Options>) {
             ? [vizConfig.facetVariable]
             : undefined,
           showMissingness: vizConfig.showMissingness ? 'TRUE' : 'FALSE',
-          returnPointIds: vizConfig.returnPointIds ?? true,
+          returnPointIds:
+            options?.returnPointIds ?? vizConfig.returnPointIds ?? true,
```

Comment:
Is this really something we want to store/control via `vizConfig`?

I think `vizConfig` should only be for visualisation state that the user can control.

I think
```suggestion
          returnPointIds:
            options?.returnPointIds ?? true,
```
should be fine (and then remove `returnPointIds` from `ScatterplotConfig`)?

Reply:
I'm happy with that rule! I'll make the changes to `ScatterplotConfig`.